srtcore/common.cpp

```diff
@@ -83,230 +83,18 @@ m_iMinor(minor)
-
-        default:
-           break;
-        }
-
-        break;
-
-      case MJ_NOTSUP:
-        m_strMsg = "Operation not supported";
- 
-        switch (m_iMinor)
-        {
-        case MN_ISBOUND:
-           m_strMsg += ": Cannot do this operation on a BOUND socket";
-           break;
-
-        case MN_ISCONNECTED:
-           m_strMsg += ": Cannot do this operation on a CONNECTED socket";
-           break;
-
-        case MN_INVAL:
-           m_strMsg += ": Bad parameters";
-           break;
-
-        case MN_SIDINVAL:
-           m_strMsg += ": Invalid socket ID";
-           break;
-
-        case MN_ISUNBOUND:
-           m_strMsg += ": Cannot do this operation on an UNBOUND socket";
-           break;
-
-        case MN_NOLISTEN:
-           m_strMsg += ": Socket is not in listening state";
-           break;
-
-        case MN_ISRENDEZVOUS:
-           m_strMsg += ": Listen/accept is not supported in rendezous connection setup";
-           break;
-
-        case MN_ISRENDUNBOUND:
-           m_strMsg += ": Cannot call connect on UNBOUND socket in rendezvous connection setup";
-           break;
-
-        case MN_INVALMSGAPI:
-           m_strMsg += ": Incorrect use of Message API (sendmsg/recvmsg).";
-           break;
-
-        case MN_INVALBUFFERAPI:
-           m_strMsg += ": Incorrect use of Buffer API (send/recv) or File API (sendfile/recvfile).";
-           break;
-
-        case MN_BUSY:
-           m_strMsg += ": Another socket is already listening on the same port";
-           break;
-
-        case MN_XSIZE:
-           m_strMsg += ": Message is too large to send (it must be less than the SRT send buffer size)";
-           break;
-
-        case MN_EIDINVAL:
-           m_strMsg += ": Invalid epoll ID";
-           break;
-
-        case MN_EEMPTY:
-           m_strMsg += ": All sockets removed from epoll, waiting would deadlock";
-
-        default:
-           break;
-        }
-
-        break;
-
-     case MJ_AGAIN:
-        m_strMsg = "Non-blocking call failure";
-
-        switch (m_iMinor)
-        {
-        case MN_WRAVAIL:
-           m_strMsg += ": no buffer available for sending";
-           break;
-
-        case MN_RDAVAIL:
-           m_strMsg += ": no data available for reading";
-           break;
-
-        case MN_XMTIMEOUT:
-           m_strMsg += ": transmission timed out";
-           break;
-
-#ifdef SRT_ENABLE_ECN
-        case MN_CONGESTION:
-           m_strMsg += ": early congestion notification";
-           break;
-#endif /* SRT_ENABLE_ECN */
-        default:
-           break;
-        }
-
-        break;
-
-     case MJ_PEERERROR:
-        m_strMsg = "The peer side has signalled an error";
-
-        break;
-
-      default:
-        m_strMsg = "Unknown error";
-   }
-
-   // Adding "errno" information
-   if ((MJ_SUCCESS != m_iMajor) && (0 < m_iErrno))
-   {
-      m_strMsg += ": " + SysStrError(m_iErrno);
-   }
-
-   return m_strMsg;
+    return getErrorMessage();
```

Comment:
It seems `CUDTException::getErrorString()` becomes unused.
Consider removing.

Reply:
It is now used in logging in groups; I'd leave it for development purposes.